src/Microsoft.AspNetCore.TestHost/TestServer.cs

```diff
@@ -27,6 +27,19 @@ public TestServer(IWebHostBuilder builder)
             _hostInstance = host;
         }
 
+        public TestServer(IWebHostBuilder builder, IFeatureCollection featureCollection)
```

Comment:
Ok, I updated it with your suggested changes. That certainly cleans up the code a lot.
I originally thought Features was an explicit interface implementation because `Features` was common word and this was to avoid conflicts with future features from other types.